packages/react-components/react-timepicker-compat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-timepicker-compat",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
The version bump happens automatically in the relase pipeline when it reads the changefile change type

Reply:
Actually why is a bump to this package necessary? nothing changes?

Reply:
I thought the official release of compat is 0.1.0. So I bumped it.

Reply:
@Hotell my compat package is public with version 0.0.1, is this considered as a stable release or should I bump it to 0.1.0? If so, am I bumping it correctly? (I updated the package.json to 0.0.2 so next release will put it to 0.1.0)

Reply:
Ah yeah it didn't register to me that it was 0.0.1. In this case we need a bump since semver will treat patch versions are 'major' versions

Reply:
cool. So I'm keeping this change.